http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeServiceDbQueries.scala

```diff
@@ -306,411 +247,267 @@ object RuntimeServiceDbQueries {
-    ownerWorkspaceIds: Set[WorkspaceResourceSamResourceId],
-    readerGoogleProjectIds: Set[ProjectSamResourceId],
-    ownerGoogleProjectIds: Set[ProjectSamResourceId]
-  ): DBIO[Vector[ListRuntimeResponse2]] = {
-    val cp = cloudProvider.map(cp => Right(cp))
-    listAuthorizedRuntimesHelper(
-      labelMap,
-      excludeStatuses,
-      creatorOnly,
-      workspaceId,
-      cp,
-      readerRuntimeIds.map(_.resourceId).toList,
-      readerWorkspaceIds.map(_.resourceId).toList,
-      ownerWorkspaceIds.map(_.resourceId).toList,
-      readerGoogleProjectIds.map(_.resourceId).toList,
-      ownerGoogleProjectIds.map(_.resourceId).toList
-    )
-  }
+    ownerGoogleProjectIds: Set[ProjectSamResourceId] = Set.empty,
+    ownerWorkspaceIds: Set[WorkspaceResourceSamResourceId] = Set.empty,
+    readerGoogleProjectIds: Set[ProjectSamResourceId] = Set.empty,
+    readerRuntimeIds: Set[SamResourceId] = Set.empty,
+    readerWorkspaceIds: Set[WorkspaceResourceSamResourceId] = Set.empty,
 
-  /**
-   * Query the runtimes (CLUSTER) table with the given filters. Only return authorized resources (per reader*Ids and/or owner*Ids).
-   * @param labelMap
-   * @param excludeStatuses
-   * @param creatorOnly
-   * @param workspaceId
-   * @param cloudContextOrCloudProvider
-   * @param readerRuntimeIds
-   * @param readerWorkspaceIds
-   * @param ownerWorkspaceIds
-   * @param readerGoogleProjectIds
-   * @param ownerGoogleProjectIds
-   * @return
-   */
-  private def listAuthorizedRuntimesHelper(
     // Filters
-    labelMap: LabelMap,
-    excludeStatuses: List[RuntimeStatus],
-    creatorOnly: Option[WorkbenchEmail],
-    workspaceId: Option[WorkspaceId],
-    cloudContextOrCloudProvider: Option[Either[CloudContext, CloudProvider]],
-
-    // Authorizations
-    readerRuntimeIds: List[String],
-    readerWorkspaceIds: List[String],
-    ownerWorkspaceIds: List[String],
-    readerGoogleProjectIds: List[String],
-    ownerGoogleProjectIds: List[String]
-  ): DBIO[Vector[ListRuntimeResponse2]] = {
-
-    // Authorize: show only resources the user is permitted to see
-    val filterVisibleIds = getFilterVisibleIds(readerRuntimeIds,
-                                               readerWorkspaceIds,
-                                               ownerWorkspaceIds,
-                                               readerGoogleProjectIds,
-                                               ownerGoogleProjectIds
-    )
-
-    // Filter: show only user selections
-    val filterWorkspaceId = workspaceId match {
-      case Some(wid) =>
-        List(s"C.`workspaceId` = '${wid.value.toString}'")
-      case None => List.empty
+    cloudContext: Option[CloudContext] = None,
+    cloudProvider: Option[CloudProvider] = None,
+    creatorEmail: Option[WorkbenchEmail] = None,
+    excludeStatuses: List[RuntimeStatus] = List.empty,
+    labelMap: LabelMap = Map.empty[String, String],
+    workspaceId: Option[WorkspaceId] = None
```

Comment:
Every param now has a default, empty value. If you call `listRuntimes` with no arguments you'll get no results.